fmn/consumer/backends/mail.py

```diff
@@ -151,9 +151,11 @@ def _format_line(msg):
         n = len(queued_messages)
         subject = u"Fedora Notifications Digest (%i updates)" % n
         summary = u"Digest summary:\n"
+        i = 0
         for msg in queued_messages:
+            i = i + 1
             line = fedmsg.meta.msg2subtitle(msg, **self.config) or u''
-            summary += str(i+1) + ".\t" + line + "\n"
+            summary += '%d.\t%s\n' % (i, line)
```

Comment:
My suggestion had more been about using ```enumerate()``` to avoid the ```i = 0``` and the ```i = i + 1```, but you can leave it this way if you like since it's functionally equivalent.